elixir/apps/web/lib/web/live/clients/index.ex

```diff
@@ -111,7 +111,7 @@ defmodule Web.Clients.Index do
           </:col>
           <:empty>
             <div class="text-center text-neutral-500 p-4">
-              No clients to display. Clients are created automatically when a user connects to a resource.
+              No clients to display. Clients are created automatically when a user signs in from an application.
```

Comment:
Similar thought as comment above:

```suggestion
              No Actors have signed in from any Client.
```

Reply:
@jamilbk since you wrote the previous version, WDYT about this suggestion?

Reply:
LGTM